src/components/footer.tsx

```diff
@@ -38,7 +37,7 @@ const navigation = {
     },
     {
       name: 'Twitter',
-      href: '#',
+      href: 'https://twitter.com/OutBoulder?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor',
```

Comment:
similar to the above comment, I'd strip off the query params here as well. href value should be `https://twitter.com/OutBoulder`